ofrak_patch_maker/ofrak_patch_maker/toolchain/utils.py

```diff
@@ -5,15 +5,19 @@
 from multiprocessing import Pool, cpu_count
 from typing import Optional, Dict, Mapping, Tuple
 
-import magic
-
 from ofrak_patch_maker.toolchain.model import BinFileType, Segment
 from ofrak_type.error import NotFoundError
 from ofrak_type.memory_permissions import MemoryPermissions
 
+_magic = None
+
 
 def get_file_format(path):
-    result = magic.from_file(path).split(" ")[0].lower()
+    global _magic
+    if _magic is None:
+        import magic as _magic
+
+    result = _magic.from_file(path).split(" ")[0].lower()
```

Comment:
I see. What about deleting lines 12-13, and something like this:

```suggestion
    try:
        import magic
    except ImportError:
        # ImportError is likely raise because libmagic cannot be found on the system. See error message.
        raise
    result = magic.from_file(path).split(" ")[0].lower()
```